### PR TITLE
Create MakeItSmall

### PR DESCRIPTION
**Pull request by labo-css**
NB : github automatically added my previous pull request "extraplanetory launchpads" to this pull request, why? Don't know how to remove it from this pull request

### CODE REVIEW

**Comment by plague006** [comment]
It's included in the PR because you had that commit in your master. For individual PRs you want to use individual branches.

**Comment by labo-css** [comment]
Thanx, should i have to make another pull request or this one is ok for now?

**Comment by plague006** [comment]
This isn't ideal, but also not important enough to make a fuss about.

**Comment by labo-css** [comment]
Thanx, the next time i contribute to this repo, i will do it better. This is really great to have this tool, CKAN is the best!

File: MakeItSmall/MakeItSmall-1.1.ckan [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    "spec_version"   : 1,
+    "name"           : "MakeItSmall",
+    "abstract"       : "MakeItSmall let your resize the NavBall, Crew, Timer and Altimeter",
+    "identifier"     : "MakeItSmall",
+    "download"       : "https://ksp.sarbian.com/jenkins/job/MakeItSmall/3/artifact/MakeItSmall-1.1.0.0.zip",
+    "version"        : "1.1",
+    "author"         : "sarbian",
+    "license"        : "MIT",
+    "release_status" : "stable",
+    "ksp_version_min"    : "1.1.0",
```
Comment (plague006) [inline MakeItSmall/MakeItSmall-1.1.ckan]: Deleting `ksp_version_min` and `ksp_version_max` and instead using `"ksp_version": "1.1"` will help us future-proof this metadata.

**Comment by labo-css** [comment]
Noted, you push it? I push it? Is there any form of list of metadata i can use in a .ckan?

**Comment by plague006** [comment]
I can't push changes into your PR because it's your local branch. Metadata fields are listed in the spec: https://github.com/KSP-CKAN/CKAN/blob/master/Spec.md

**Comment by labo-css** [comment]
Ok then, i re-do it

**Comment by labo-css** [comment]
Sorry, i just tried to make a new branch and did it but... the launchpad plugin show up again, i'm very new to git, can you do it for me?

https://github.com/labo-css/CKAN-meta/blob/labo-css-makeitsmall/MakeItSmall/MakeItSmall-1.1.ckan

I just don't understand where i fuckup

**Comment by plague006** [comment]
I found git via github really difficult. I had a much easier time using smartgit (http://www.syntevo.com/smartgit/), it just made more sense to me.

If you're using the github web UI though, the easiest way to submit new files is probably to do it via the KSP-CKAN-owned repo. So I'll give a few screenshots of me submitting to your fork -- when you submit to our repo it'll be the same except you'll be at https://github.com/KSP-CKAN/CKAN-meta/ instead of https://github.com/labo-css/CKAN-meta/

1. Hit the "new file" button http://puu.sh/orsaW/593c24b637.png http://puu.sh/orsev/79339091fc.png
2. Enter the file name and info http://puu.sh/orska/4f1958c134.png
3. Give a description of what the change is and then his "Propose New File" http://puu.sh/orsmU/0c1dcbce59.png
4. Click "Create Pull Request" http://puu.sh/orsot/e4e1b2f545.png
5. Add any additional info then hit "Create Pull Request" again http://puu.sh/orsr2/03ce76f695.png

**Comment by plague006** [comment]
Lastly I'm going to ping @politas since I believe he uses the GitHub web UI and might have extra info or corrections to make to the info I provided.

**Comment by politas** [comment]
I use a mix of Github web interface and a local git clone on my computer.
If you commit a change to your master branch, that's always going to cause issues for you. Ideally, make a branch, add or modify files in that branch, then create a pull request from that branch. If you have two changes applied to your master and can't figure out how to return to being in sync with the upstream, the easiest way would be to generate a pull request from your master to the KSP-CKAN master, including both changes.

The trickier and less elegant way would be to make local copies of your changed files, delete your fork of the KSP-CKAN repo and create a fresh fork, then create individual branches for your changes as @plague006 mentioned above.

If you wish to use git locally, feel free to jump into #ckan IRC channel on espernet and ask me questions. I'm not an expert on git, yet, but I'm starting to get a hold of how it all works.

**Comment by labo-css** [comment]
Thanx everybody.

**Pull request closed without merging**